services/src/handlers/datasets.rs

```diff
@@ -1180,15 +1219,16 @@ mod tests {
             auto_detect_meta_data_definition(test_data!("vector/data/points_with_iso_time.json"))
                 .unwrap();
 
-        if let MetaDataDefinition::OgrMetaData(meta_data) = &mut meta_data {
+        if let crate::datasets::storage::MetaDataDefinition::OgrMetaData(meta_data) = &mut meta_data
+        {
             if let Some(columns) = &mut meta_data.loading_info.columns {
                 columns.datetime.sort();
             }
         }
 
         assert_eq!(
             meta_data,
-            MetaDataDefinition::OgrMetaData(StaticMetaData {
+            crate::datasets::storage::MetaDataDefinition::OgrMetaData(StaticMetaData {
```

Comment:
shouldn't this be the MetaDataDefinition from services/api/model ?

Reply:
I thought of `auto_detect_meta_data_definition` more of like an internal utility function which isn't directly connected to the API. Because of that I used internal types.